### PR TITLE
support removing table_id/store_id from namespace.

### PR DESCRIPTION
**Pull request by dantin**
1. add info log
2. add function to remove table_id/store_id from namespace

### CODE REVIEW

File: pdctl/command/namespace_command.go [hunk 1]
```diff
@@ -86,11 +97,31 @@ func appendTableCommandFunc(cmd *cobra.Command, args []string) {
...
+		return
 	}
 
 	input := make(map[string]interface{})
 	input["namespace"] = args[0]
 	input["table_id"] = args[1]
+	input["action"] = "remove"
 
-	postJSON(cmd, namespaceAddPrefix, input)
+	postJSON(cmd, namespaceRelPrefix, input)
```
Comment (disksing) [inline pdctl/command/namespace_command.go]: Why use another URL as you have defined different action fields?

File: server/api/namespace.go [hunk 2]
```diff
@@ -97,10 +98,27 @@ func (h *namespaceHandler) Append(w http.ResponseWriter, r *http.Request) {
...
+	if !ok {
+		h.rd.JSON(w, http.StatusBadRequest, errors.New("missing parameters"))
+		return
+	}
 
-	// append table id to namespace
-	if err := cluster.AddNamespaceTableID(ns, tableID); err != nil {
-		h.rd.JSON(w, http.StatusInternalServerError, err.Error())
+	switch action {
+	case "append":
```
Comment (ming-relax) [inline server/api/namespace.go]: How about `add` instead of `append`?

File: pdctl/command/namespace_command.go [hunk 3]
```diff
@@ -86,11 +97,31 @@ func appendTableCommandFunc(cmd *cobra.Command, args []string) {
 	}
 	if _, err := strconv.Atoi(args[1]); err != nil {
 		fmt.Println("table_id shoud be a number")
+		return
+	}
+
+	input := make(map[string]interface{})
```
Comment (siddontang) [inline pdctl/command/namespace_command.go]:
```go
input := map[string]interface{} {
    "namespace" : args[0],
    ......
}
```

may be better.

File: pdctl/command/store_command.go [hunk 4]
```diff
@@ -30,14 +30,15 @@ var (
 // NewStoreCommand return a store subcommand of rootCmd
 func NewStoreCommand() *cobra.Command {
 	s := &cobra.Command{
-		Use:   "store [delete|label|weight] <store_id>",
+		Use:   "store [delete|label|weight|set_namespace|rm_namespace] <store_id> [namespace]",
```
Comment (siddontang) [inline pdctl/command/store_command.go]: I prefer using another subcommand here, not in store. maybe `store_ns set|rm`

File: pdctl/command/namespace_command.go (hunk 1 above)
Comment (dantin) [reply pdctl/command/namespace_command.go]: The old URL may confuse user as it also support 'delete' function.

File: server/api/namespace.go (hunk 2 above)
Comment (dantin) [reply server/api/namespace.go]: OK

File: pdctl/command/namespace_command.go (hunk 3 above)
Comment (dantin) [reply pdctl/command/namespace_command.go]: OK, I'll fix it.

File: pdctl/command/store_command.go (hunk 4 above)
Comment (dantin) [reply pdctl/command/store_command.go]: OK, I'll move it to another subcommand.

File: pdctl/command/namespace_command.go [hunk 5]
```diff
@@ -23,18 +23,19 @@ import (
 
 const (
 	namespacePrefix    = "pd/api/v1/namespaces"
-	namespaceAddPrefix = "pd/api/v1/namespaces/add"
+	namespaceRelPrefix = "pd/api/v1/namespaces/rel"
```
Comment (siddontang) [inline pdctl/command/namespace_command.go]: I am confused what dos `rel` mean here?

Comment (dantin) [reply pdctl/command/namespace_command.go]: What I means here is relationship. @disksing raised the same question, maybe we need a new name to make it more clear.

Comment (siddontang) [reply pdctl/command/namespace_command.go]: I think you can use `/namespaces/table` here.

**Comment by siddontang** [comment]
PTAL @disksing

File: pdctl/command/namespace_command.go [hunk 6]
```diff
@@ -22,19 +22,20 @@ import (
...
-	namespaceAddPrefix = "pd/api/v1/namespaces/add"
+	namespacePrefix      = "pd/api/v1/namespaces"
+	namespaceTablePrefix = "pd/api/v1/namespaces/table"
 )
 
 // NewNamespaceCommand return a namespace sub-command of rootCmd
 func NewNamespaceCommand() *cobra.Command {
 	s := &cobra.Command{
-		Use:   "namespace [create|append]",
+		Use:   "namespace [create|add|remove]",
```
Comment (siddontang) [inline pdctl/command/namespace_command.go]: Seem that create is to create the namespace but add/remove is to operate the table namespace. I think it is not proper to put these commands together.

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
LGTM.

**Pull request merged**